lessons/Data-Science/box-plots-interpret/langs/en-us/index.adoc

```diff
@@ -0,0 +1,384 @@
+
+| *Histograms*	
+| Fixed Bins      
+| Variable. Points “pile up in bins” +
+so we can see how many are in each.
+| A long tail of +
+shorter bars to one side
+|===
+
+
+@slidebreak
+
+
+
+=== Investigate
+
+@teacher{
+
+The activity described below is a *continuation* of an activity launched during @lesson-link{histograms-visualize}.
+
+During that activity, students rotated around the room in teams of 3 to posters that were labeled "Skew Left", or "Skew Right", and "Symmetric". At each poster, groups were directed to (1) sketch a histogram of the given distribution, and then (2) describe a dataset or scenario that would produce that data visualization.
+
+A third section of each poster was left blank. During the subsequent activity, students will complete that third section.
+}
+
+@lesson-instruction{
+- Your teacher has instructed you and your group to stand in front of a poster that says "Symmetric", "Skew Left", or "Skew Right" at the top.
+- The paper in front of you should have a histogram and a description of a dataset / scenario that would produce the given distribution.
+- In the bottom section of the poster, sketch the *box plot* that would result in the distribution to match the histogram and scenario.
+}
+
+@teacher{
+Spend some time debriefing with students, using the prompts below. Responses will vary for each group of students.
+}
+
+@QandA{
+@Q{What strategies did you use to make sure that your histogram and box plots matched?}
+
+@Q{Did all of the box plots your classmates developed correctly represent the given scenario?}
+
+@Q{What information does a box plot convey that a histogram doesn't?}
+
+@A{What information does a histogram convey that a box plot doesn't?}
+
+}
+
+
+@lesson-instruction{
+- Use what you've learned about how box plots and histograms display data differently to complete @printable-exercise{matching-boxplots-to-histograms.adoc}.
+
+- @opt{Complete @opt-printable-exercise{matching-boxplots-to-histograms-2.adoc} and/or the Matching Box Plots to Histograms slide of @opt-starter-file{boxplots}}
+}
+
+@clear
+@slidebreak
+
+@lesson-instruction{
+Now that you have the skills to interpret box plots, complete @printable-exercise{data-cycle-box-plots-animals.adoc}.
+}
+
+@pd-slide{
+*Matching Debrief: Think about shape, don't rely on numbers!*
+
+There are teachers and students who complain about this page. They say that this page is unfair because we didn't label the axes. The stats teachers are always shocked at their colleagues. They say, "It doesn't matter! We're talking about shape. Skew right is skew right, no matter where it falls on the x axis.""
+
+So many kids have sort of a statistical lazy eye. How do you fix that? You wear a patch over the strong eye. By removing the numbers, it forces people to think about the shape. If the numbers are there, you get high achieving algebra students who never learn shape because they keep using the numbers.
+}
+
+
+
+=== Synthesize
+
+@QandA{
```

Comment:
I like this questions and would like to see it in the lesson, perhaps as an opt-printable teachers could give for homework? But, to me, a synthesize questions gets at bigger ideas, eg.

- How do dense data clusters show up differently on histograms and box plots?
- What features of a dataset are easier to identify on a histogram?
- What features of a dataset are easier to identify on a box plot?

Reply:
scrubbing the question for now; can try to find it a home later. I like the idea of using it for MDBook quiz!